### PR TITLE
feat: :boom: Added more items to the in respect to issue #6

### LINKED ISSUE #6: Adding more books in the app
Currently there are a minimum of just 5, we need more atleast a 10-12 to showcase.

### PR DESCRIPTION
**Pull request by Omafovbe**
**Description**
Added images and data for more books and updated the cart function to source images from public directory. Also, dynamically add the items to the home  component

This PR fixes #6
- Having just a few items for display to add to cart
- dynamically getting the image to display
- a little change but spaced the cart text from the total number of items in cart
- 
**Notes for Reviewers**

### CODE REVIEW

File: src/data/books.json [hunk 1]
```diff
@@ -0,0 +1,202 @@
+[
```
Comment (kamalbuilds) [inline src/data/books.json]: I appreciate this moving to a new .json file helps in simplifing :100:

File: src/data/books.json [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+            "imgUrl": "javascript.jpg",
+            "numbers": 0,
+            "inCart": false
+        
+    },
+    {
+        
+            "name": "jQuery",
+            "tagName": "jquery",
+            "imgUrl": "jquery.png",
```
Comment (kamalbuilds) [inline src/data/books.json]: how are these imgurl working ? without giving full http:.... url?

File: src/reducers/basketReducer.js [hunk 3]
```diff
@@ -1,66 +1,20 @@
...
 }
 
 export default (state = initialState, action) => {
     let productSelected = "";
+    // let productId = "";
     switch(action.type) {
         case ADD_PRODUCT_BASKET:
-            productSelected = { ...state.products[action.payload]}
+            // productSelected = { ...state.products[action.payload] }
+            productSelected = { ...state.books.find(book => book.tagName === action.payload) };
```
Comment (kamalbuilds) [inline src/reducers/basketReducer.js]: This is wondurful to add the new logic but At all other places we are using action.payload as arg , just for consistency comment the new logic and uncomment the older one .

**Review by kamalbuilds — APPROVED** [review]
The changes have been very Impressive :1st_place_medal:
 just to look at these minor details.

**Comment by Omafovbe** [comment]
Thanks for the comments.

1. The imgurl for now are the filenames of the image stored in the
public folder and are referenced from home and cart component using
JavaScript literal as thus `.. images/${book.imgurl}` which can be easily
updated show Incase we need to pull the image from a remote url in the
future

2. For the basketReducer.js firstly we need to get all of the items in the
JSON file and that is why I made a find call to the item database with the
action.payload arg. Apart from that every other code is consistent.

Thanks qgain

**Comment by kamalbuilds** [comment]
thanks @Omafovbe  these details would be helpful to be put up in README.

**Comment by kamalbuilds** [comment]
Hey @Omafovbe were these changes merged to the main branch :thinking:  I dont see them

**Comment by Omafovbe** [comment]
You didn't merge to the main branch but merge to a new branch in the repo.
Check

**Comment by kamalbuilds** [comment]
@Omafovbe I dont see it there too https://github.com/legendarykamal/Lets-go-shop/compare/new...master

Better please open a new PR for merging on the master branch .

**Comment by kamalbuilds** [comment]
I am eager to see your name in the contribitors list   :smile:

**Comment by kamalbuilds** [comment]
also @Omafovbe revert your last commit done on aug 21 https://github.com/legendarykamal/Lets-go-shop/compare/master...Omafovbe:Lets-go-shop:add-more-items 
by this command on the ```add to item branch```
```
git reset --soft HEAD~1
``` 
and then open a PR against the master branch 

as I dont know why you have merged your master branch to ur add to item branch only as per this https://github.com/legendarykamal/Lets-go-shop/compare/master...Omafovbe:Lets-go-shop:add-more-items

**Comment by Omafovbe** [comment]
Done, reset the branch and opened a new PR. Thanks for the opportunity

**Pull request merged**